docs/en/advanced/Alluxio-Storage-Management.md

```diff
@@ -12,22 +12,50 @@ priority: 0
 
 ## Alluxio Storage Overview
 
-TODO(Zac): Add an overview of Alluxio storage here, covering
+
+<!-- ################## -->
+<!-- TODO(Zac): Add an overview of Alluxio storage here, covering
 - relationship and difference between Alluxio storage and under storage.
     * replication: hot data in Alluxio storage can have mulitple copies
     * once data is persisted, data can have zero copy in Alluxio but still available to
    applications.
-- the indication to application I/O performance when data is cached in Alluxio storage.
-
-Alluxio manages the local storage, including memory, of Alluxio workers to act as a distributed
-buffer cache. This fast data layer between user applications and the various under storages results
-in vastly improved I/O performance.
+- the indication to application I/O performance when data is cached in Alluxio storage. -->
+<!-- ################## -->
+
+Alluxio helps unify users' data across a variety of platforms while also helping to increase
+overall I/O throughput from a user's perspective. Alluxio accomplishes this by splitting storage
+into two distinct categories
+
+- **UFS (Under Filesystem, also sometimes called under storage)**
+    - This storage is represented as space which is not managed by Alluxio. UFS storage may come
+    from an external system such as HDFS, S3, or any other type of filesystem. Alluxio
+    may connect to one or more of these UFSs and expose them within a single namespace. This is one
+    of the main features of Alluxio.
+- **Alluxio storage**
+    - Alluxio manages the local storage, including memory, of Alluxio workers to act as a
+    distributed buffer cache. This fast data layer between user applications and the various under
+    storages results in vastly improved I/O performance.
+    - The amount and type of storage for each Alluxio node to manage is determined by user
+    configuration.
+
+
+<!-- Will eventually get an image specifically for docs, but for now will use this as reference -->
+![Alluxio storage diagram](https://www.alluxio.org/images/alluxio-unifies-access.png)
+
+One of the reasons that Alluxio storage improves performance is that it stores data in memory
```

Comment:
can we state the relationship between the under storage and Alluxio storage, from user's perspective. To name a few:
- Achieving different goals for users: Alluxio storage stores hot and transient data, under storage provides persistency
- If data is in under storage, even it is not loaded into Alluxio yet, it is visible to Alluxio clients, and Alluxio will automatically load the recently accessed data and cache it. 
- Different replication scheme (you already covered)